docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -33,46 +33,18 @@ DaemonSet has two update strategy types :
   DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
   will be created automatically, in a controlled fashion. 
 
-## Limitations 
-
-* DaemonSet rollout history is not supported yet. 
-* DaemonSet rollback is not directly supported in `kubectl` yet. You can rollback
-  by updating DaemonSet template to match the previous version.
-
 ## Caveat: Updating DaemonSet created from Kubernetes version 1.5 or before 
 
 If you try to rolling update a DaemonSet that was created from Kubernetes
```

Comment:
While you're at it, can you change this to:
"If you try a rolling update on a DaemonSet that was..."